src/main/java/de/chojo/repbot/ReputationBot.java

```diff
@@ -128,11 +132,24 @@ private void start() throws SQLException, IOException {
     private void initBotList() {
         var botlist = configuration.botlist();
         if (!botlist.isSubmit()) return;
-        BotlistReporter.build(shardManager)
+        BotlistService.build(shardManager)
                 .forDiscordBotListCOM(botlist.discordBotlistCom())
                 .forDiscordBotsGG(botlist.discordBotsGg())
                 .forTopGG(botlist.topGg())
+                .forBotlistMe(botlist.botListMe())
                 .withExecutorService(repBotWorker)
+                .withVoteService(builder -> builder
+                        .withVoteWeebhooks("localhost", 8000)
```

Comment:
Sure, you want to listen on localhost and not a public interface?

Reply:
These values should be set in the config anyway.
I added them in https://github.com/RainbowDashLabs/reputation-bot/pull/136/commits/7b290362aba06018ea6136511c6b2bcde036098d